### ISSUE TITLE
Show some form of success message when receiving payment in receive screen

### ISSUE THREAD

**Issue by holzeis** [get10101/10101#882]
When creating an invoice and the user waits on the receive screen, the wallet should show confetti once the payment is claimed. By acknowledging the payment the user should be navigated back to the home screen (wallet history).

**Comment by klochowicz** [comment]
🤔 we added this to the trades,  as they are the special feature of the wallet, the reason being that we try to encourage them by showing the festiveness. 
by showing sparkles on every transaction, that will have less effect - or at least it should have *bigger* effect on the trading screen. Rainbows perhaps?L  

I, for one, don't mind rainbows and unicorns being everywhere 🌈🦄 ☺️

**Comment by holzeis** [comment]
I am not committed to the confettis. If you can make a better animation with rainbows and unicorns then go for it. 🦄🌈

**Comment by klochowicz** [comment]
confettis are awesome, don't get me wrong, I was just trying to see why you wanted to have them outside of trade context - and pointing out that we'd need to make the trades stand out again from normal wallet activity, should we adopt confettis everywhere.

for me it's really simple: you want to see more confetti -> trade more 🫠

**Comment by holzeis** [comment]
Maybe I haven't been clear on the ticket description, but the important thing is about showing that you have received the payment while you are at the receive screen.

Breez, Zeus, Phonex - all wallets I know of do that.

**Comment by bonomat** [comment]
I'm for unicorns and rainbows: 

![image](https://github.com/get10101/10101/assets/224613/e7ad5962-a493-48e5-addd-f63d74063be5)

**Comment by klochowicz** [comment]
@holzeis can we change the title then to remove the "confetti" from the scope? letting the user know that they received payment is an important feature.

**Comment by holzeis** [comment]
> @holzeis can we change the title then to remove the "confetti" from the scope? letting the user know that they received payment is an important feature.

@klochowicz I changed the title.. is it better now?

**Comment by klochowicz** [comment]
> > @holzeis can we change the title then to remove the "confetti" from the scope? letting the user know that they received payment is an important feature.
> 
> @klochowicz I changed the title.. is it better now?

yes, that's awesome! I fully agree that we should do this soon in the wallet scope. thank you 🙏

**Comment by klochowicz** [comment]
this does seem like a duplicate of #180  - is there anything more in this ticket @holzeis or can we close it in favour of the other one?

**Comment by holzeis** [comment]
Yes that's a duplicate. Thanks for finding it 🙂
